src/Compilers/CSharp/Test/Symbol/Symbols/Metadata/PE/LoadingFields.cs

```diff
@@ -140,5 +140,18 @@ static void Main()
             CompileAndVerify(compilation, expectedOutput: @"Value1
 Value2");
         }
+
+        [Fact]
+        public void TestLoadFieldsOfReadOnlySpanFromCorlib()
+        {
+            var comp = CreateCompilation("", targetFramework: TargetFramework.Net60);
+
+            var readOnlySpanType = comp.GetTypeByMetadataName("System.ReadOnlySpan`1");
```

Comment:
>GetTypeByMetadataName [](http://example.com/codeflow?start=40&length=21)

I think we have a ```GetSpecialType``` API. It looks like the two following asserts won't be necessary then.